pkg/tanka/parse.go

```diff
@@ -140,7 +140,7 @@ func evalJsonnet(baseDir string, env *v1alpha1.Config, extCode map[string]string
 		return nil, err
 	}
 
-	var dict map[string]interface{}
+	var dict interface{}
```

Comment:
```suggestion
	var data interface{}
```

Given it's not a dict anymore, let's call it data :)